cloudhub-2/modules/ROOT/pages/_partials/deployment.adoc

```diff
@@ -96,6 +96,29 @@ To configure additional log levels and categories to include in logs:
 // end::configureLogging[]
 
 
+// tag::enableDistributedTracing[]
+
+== Enable Trace Data Collection
+
+Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into all APM solutions.
```

Comment:
1. Can we expand APM  -- is this Anypoint Monitoring?
2. The only way CH2 customers can leverage Traces today is by using the Telemetry Exporter feature. Saying all APM solutions can lead to customers believing there is more they can do with traces. Can we rephrase as:

`Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into the Anypoint Monitoring Telemetry Exporter <link to Telemetry Exporter docs>
`

Reply:
APM = Anypoint Partner Manager

Reply:
Rephrased as requested